src/pages/Notifications/EventLog/EventLogPage.tsx

```diff
@@ -123,18 +123,19 @@ export const EventLogPage: React.FunctionComponent = () => {
     async (id: UUID) => {
       const query = getEndpoint.query;
       const endpoint = await query(id);
-      if (endpoint.payload?.type === 'Endpoint') {
+      if (endpoint.payload?.type === 'EndpointDTO') {
```

Comment:
Same as above, is this DTO needed?
```suggestion
      if (endpoint.payload?.type === 'Endpoint') {
```